src/web/routes/application/middleware/handle-path-request/handle-path-request.js

```diff
@@ -18,13 +19,16 @@ const middleware = (config, pathsInSequence, step) => (req, res, next) => {
     return next()
   }
 
-  // Initialise nextAllowedStep if none exists in session
-  if (!req.session.nextAllowedStep) {
-    req.session.nextAllowedStep = pathsInSequence[0]
+  // Initialise nextAllowedPath if none exists in session
+  const firstPathInSequence = pathsInSequence[0]
```

Comment:
This `const` might make more sense inside the if statement below

Reply:
Updated